tests/cupy_tests/random_tests/common_distributions.py

```diff
@@ -127,6 +127,23 @@ def _check_ks(
             raise AssertionError(message)
 
 
+uniform_params = [
+    {'low': 1, 'high': 10.0, 'size': (3, 5)},
+    {'low': cupy.array([1, 2]), 'high': 3, 'size': None},
```

Comment:
Could you move cupy.array creation into `test_uniform` to avoid GPU memory allocation outside of the test? (see `Beta.test_beta`)
```suggestion
    {'low': [1, 2], 'high': 3, 'size': None},
```

Reply:
Fixed